microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-106.md

```diff
@@ -0,0 +1,182 @@
+---
+title: What's new in DevTools (Microsoft Edge 106)
+description: "Introducing the Command Palette. Filter and automatically fix code issues in DevTools for Visual Studio Code. DevTools now uses less disk space when caching sourcemaps. Chromium color themes have been deprecated. Accessibility improvements in Focus Mode and high contrast mode. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/28/2022
+---
+# What's New in DevTools (Microsoft Edge 106)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Introducing the Command Palette
+
+<!-- Subtitle: Your favorite commands in one place. -->
+<!--           Your favorite browser and DevTools commands in one place. -->
+
+Command Palette makes it possible to access browser management and developer tools with just one keyboard shortcut: `Ctrl`+`Q`.  Let us know what commands we should add next!
+
+To list the available browser management commands and DevTools commands, press `Ctrl`+`Q`.  To filter the commands, start typing:
```

Comment:
"To access the available browser..." instead of using the word "list," since commands don't appear without input.

Reply:
changed